.changeset/catalog-import-integration-config-sunny-london.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-catalog-import': minor
```

Comment:
With the suggestion below we'll be able to make this a `patch` change as it won't be a breaking change